### PR TITLE
SDI-88: ♻️  Move out of app.js into index and standard router

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: server/authentication/auth.ts [hunk 1]
```diff
@@ -14,6 +17,24 @@ passport.deserializeUser((user, done) => {
   done(null, user as Express.User)
 })
 
+export type AuthenticationMiddleware = () => RequestHandler
+
+const authenticationMiddleware: AuthenticationMiddleware = () => {
```
Comment (petergphillips) [inline server/authentication/auth.ts:22]: copied same pattern as in typescript template

File: server/routes/testutils/appSetup.ts [hunk 2]
```diff
@@ -42,5 +43,6 @@ export default function appWithAllRoutes({
   production?: boolean
   hmppsAuthMFAUser?: boolean
 }): Express {
+  auth.default.authenticationMiddleware = () => (req, res, next) => next()
```
Comment (petergphillips) [inline server/routes/testutils/appSetup.ts:46]: copied same pattern as in typescript template

File: server/authentication/auth.ts (hunk 1 above)
Comment (steverendell) [reply server/authentication/auth.ts:22]: Ah this might mean the return url might actually start working !

**Review by steverendell — APPROVED** [review]
lgtm

**Pull request merged**